src/App.scss

```diff
@@ -56,3 +56,9 @@
 #cvss-vector-content {
     font-size: var(--pf-global--FontSize--sm);
 }
+
+#cves-severity-question-mark {
+    color: var(--pf-global--secondary-color--100);
+    vertical-align: -0.2em;
+    margin-left: 16px;
```

Comment:
Nit-pick: you can provide the color value directly in the icon component with `color` prop. Also for the alignment stuff like margin and align: I would recommend using PF's `Flex` component pr smth similar. This will let you avoid writing custom CSS and also provide more confidence regarding responsiveness of your components

Reply:
Thank you, for review, I tried to keep all styles in one place instead of having some styles in .js and some in .scss. This style is for column header which should not be responsive (we don't want the icon to be moved to the next line, do we), but generally yes Flex is better.